### PR TITLE
Fixes functional test C2660, CC14407

### PR DESCRIPTION
**Pull request by garyr**
## Description
Stops using the alloyEvent helper in C2660, C14407

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have submitted a [documentation](https://github.com/AdobeDocs/alloy-docs) pull request or no changes are needed.
- [x] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

File: test/functional/specs/C14407.js [hunk 1]
```diff
@@ -35,22 +34,29 @@ const setConsentIn = ClientFunction(() => {
   return window.alloy("setConsent", { general: "in" });
 });
 
+const sendEventWithDelay = ClientFunction((data, delay) => {
```
Comment (jonsnyder) [inline test/functional/specs/C14407.js:37]: Should we make this a helper?  Also can you write a comment on why the delay is necessary and under what circumstances it should be used?

File: test/functional/specs/C2660.js [hunk 2]
```diff
@@ -23,6 +22,19 @@ const setConsentIn = ClientFunction(() => {
...
+ * registered, but not block on waiting for consent promise to resolve
+ */
+const sendEventWithDelay = ClientFunction((data, delay) => {
+  return new Promise(resolve => {
+    window.alloy("event", data);
+    setTimeout(() => {
+      resolve();
+    }, delay || 100);
+  });
+});
```
[the remark this answers is not in the document]
Comment (Aaronius) [reply test/functional/specs/C2660.js:36]: Ooh! Actually, I just noticed we have a `flushPromiseChains` helper in `test/functional/helpers/flushPromiseChains.js`. I think we can get rid of the timeout if we use it.

I think your test would look something like the following. I'm shooting from the hip here so pardon any errors.

```
const sendEvent = ClientFunction(() => {
  window.alloy("event");
});

const changeHashAndConsent = ClientFunction(() => {
  window.location.hash = "foo";
  window.alloy("setConsent", { general: "in" });
});

// Then down inside the test...

await sendEvent();
await flushPromiseChains();
await changeHashAndConsent();
await sendEvent();

// Then check the network requests.
```

**Comment by Aaronius** [comment]
Hey Gary, for future pull requests, do you mind creating the branch on the main repo instead of on a fork? It makes it easier for other team members to check out your branch, test it out, and contribute. If you need rights, see https://github.com/adobe/alloy#requesting-write-access-adobe-employees. ❤️

**Comment by jfkhoury** [comment]
@garyr I think we should reopen this PR after you've requested write rights because now the Sause labs bridge is not connecting.

**Pull request closed without merging**